com.unity.ml-agents.extensions/Runtime/Sensors/ArticulationBodySensor.cs

```diff
@@ -0,0 +1,74 @@
+#if UNITY_2020_1_OR_NEWER
+using UnityEngine;
+using Unity.MLAgents.Sensors;
+
+namespace Unity.MLAgents.Extensions.Sensors
```

Comment:
I'll probably remove this from the initial PR, but this is how the hierarchy util will be used. There will be a similar Sensor impl for RigidBodies.